frappe/public/js/frappe/form/form.js

```diff
@@ -44,7 +44,7 @@ frappe.ui.form.Form = class FrappeForm {
 	setup_meta() {
 		this.meta = frappe.get_doc('DocType', this.doctype);
 
-		if(this.meta.istable) {
+		if (this.meta.istable) {
```

Comment:
@shariquerik can you undo these (unrelated) formatting changes? Since It'll cause issues during backport.